modules/serial/serial_rn2483_radio.py

```diff
@@ -369,7 +360,8 @@ def set_rx_mode(self, message_q: queue.Queue):
         # if radio has not been put into rx mode
         if not success:
             print('error putting radio into rx mode')
-            return -1
+            self.ser.close()
```

Comment:
May not be necessary to close the serial connection if the radio cannot be set to rx, since the radio may just busy